guides/release/accessibility/page-template-considerations.md

```diff
@@ -15,14 +15,76 @@ Consider this format:
 
 Note that the unique page title is first. This is because it is the most important piece of information from a contextual perspective. Since a user with a screen reader can interrupt the screen reader as they wish, it introduces less fatigue when the unique page title is first, but provides the additional guidance if it is desired. 
 
-Until Ember provides this functionality by default, there are a few different Ember addons that will help:
+One simple way to add page titles is to create a `title` helper:
+
+```javascript {data-filename=app/helpers/title.js}
+import Helper from '@ember/component/helper';
+
+export default class Title extends Helper {
+  original = document.title;
+
+  compute([title]) {
+    document.title = title;
+  }
+
+  willDestroy() {
+    document.title = this.original;
+  }
+}
+
```

Comment:
Hard to apply in GitHub suggestion, but copy edit suggestion - delete blank line before ```